packages/assistify-ai/server/methods/SmartiReload.js

```diff
@@ -0,0 +1,154 @@
+/* globals SystemLogger, RocketChat */
+
+import {SmartiProxy, verbs} from '../SmartiProxy';
+
+Meteor.methods({
+	triggerFullResync() {
+
+		const query = {$or: [{outOfSync: true}, {outOfSync: {$exists: false}}]};
+
+		query.t = 'r';
+		const requests = RocketChat.models.Rooms.model.find(query).fetch();
+		SystemLogger.debug('Number of Requests to sync: ', requests.length);
+		for (let i=0; i < requests.length; i++) {
+			Meteor.defer(()=>Meteor.call('tryResync', requests[i]._id));
```

Comment:
This can be quite a number of planned async tasks. I'd expect we need to plan them blockwise. If not now, we'll probably have to add this lateron. But let's leave it like that until we see some issues.